html.go

```diff
@@ -44,6 +44,7 @@ const (
 	SmartypantsDashes                             // Enable smart dashes (with Smartypants)
 	SmartypantsLatexDashes                        // Enable LaTeX-style dashes (with Smartypants)
 	SmartypantsAngledQuotes                       // Enable angled double quotes (with Smartypants) for double quotes rendering
+	SmartypantsQuotesNBSP                         // Enable "French guillemets" (with Smartypants)
```

Comment:
Would it be a good idea to use French guillemets here, instead of double quotes? Go source code (and docs) are okay with unicode.